openhands/app_server/sandbox/remote_sandbox_service.py

```diff
@@ -790,7 +790,7 @@ async def inject(
         # This is primarily used for local development rather than production
         config = get_global_config()
         web_url = config.web_url
-        if web_url is None:
+        if web_url is None or 'localhost' in web_url:
```

Comment:
This is to fix polling of runtimes when running saas locally (the `web_url` does exist but its `localhost` which should qualify for polling)

We poll runtimes instead of receiving webhook events when running SAAS locally